krabs/krabs/trace.hpp

```diff
@@ -205,6 +236,8 @@ namespace krabs {
         size_t buffersRead_;
         uint64_t eventsHandled_;
 
+        EVENT_TRACE_PROPERTIES properties_;
```

Comment:
I might be wrong, but if you initialize this with ` = {};` you may be able to skip the `ZeroMemory()` calls below?

Reply:
Easy fix. I was trying to be consistent with the code in the following function.